### DISCUSSION TITLE
Encoder Decoder for a 3d mesh

### DISCUSSION THREAD

**Discussion by ck-amrahd** [facebookresearch/pytorch3d#726] [General]
Hi guys,
I want to train a model that can encode 3d mesh into latent space [fixed dimensions] and decode it to minimize the reconstruction error [similar to encoder-decoder architecture for 2d images]. I can use conv2d and TransposeConv2d for 2d images. I think I can use the GraphConv layer by transforming mesh into a graph but I am not sure how to map these variable numbers of vertices to a fixed dimensional latent space and not sure how to do the reverse [latent space to mesh] in the decoder. Any help, links, and suggestions will be helpful. Thank you.

**Comment by boqian-li** [comment]
hi, have u made any progress on this idea?
